app/views/users/show/_timeline.html.erb

```diff
@@ -3,40 +3,40 @@
     <h3 class="title is-3">Your contributions</h3><br/>
     <table class="is-narrow is-fullwidth">
       <tbody>
-        <% scoring_pull_requests.map do |pr| %>
-          <%= render partial: 'users/show/pr/pr', locals: { pr: pr } %>
-        <% end %>
-
-        <% if show_timer %>
+        <% if show_congratulations %>
```

Comment:
Is there a reason why we're now showing this congrats message above the scoring PRs?

I think it'd make far more sense to keep it rendering between the scoring & non_scoring PRs, especially as once the user has "won" and the scoring PRs is a frozen array, they might not be in chronological order with the non_scoring PRs -- so having the message to split the timeline makes sense?

![image](https://user-images.githubusercontent.com/12371363/95654359-6903f900-0af7-11eb-810d-5690de8e3786.png)